spec.bs

```diff
@@ -232,7 +232,161 @@ to conduct an auction to select an advertisement to display to the user, they ca
 {{Window/navigator}}.{{Navigator/runAdAuction()}} function, providing an auction configuration that
 tells the browser how to conduct the auction and which on-device recorded interests are allowed to
 bid in the auction for the chance to display their advertisement.
-  
+
+<h3 id="script-runners">Script Runners</h3>
+
+<pre class="idl">

```

Comment:
The linking error you were getting smells like a bikeshed bug. I filed https://github.com/speced/bikeshed/issues/2517, but for now we can manually clean this up a bit. Please `git apply` this diff: https://gist.github.com/domfarolino/c3bed9efffdeb49a1bf74db7365815d1